src/wrap.ts

```diff
@@ -28,10 +27,14 @@ export function renderWrap(element: HTMLElement, s: HeadlessState, relative: boo
   for (const c of colors) element.classList.toggle('orientation-' + c, s.orientation === c);
   element.classList.toggle('manipulable', !s.viewOnly);
 
-  const helper = createEl('cg-helper');
-  element.appendChild(helper);
   const container = createEl('cg-container');
-  helper.appendChild(container);
+  const bounds = element.getBoundingClientRect();
+  const ratio = bounds.height / bounds.width;
+  const width = Math.floor(bounds.width / 8) * 8;
+  const height = width * ratio;
+  container.style.width = width + 'px';
+  container.style.height = height + 'px';
+  element.appendChild(container);
```

Comment:
Maybe factor out the code that's repeated in `updateBounds()`?

Reply:
Done